tests/README.md

```diff
@@ -0,0 +1,10 @@
+## Instructions on how to run the terraform modules tests
+
+1. The Terraform version should be the latest (i.e. v1.9.0): this is advised because on earlier versions, tests were at beta stage.
+2. Through terminal, set up the following environment variables: 
+    -  `export MONGODB_ATLAS_PUBLIC_KEY="<YOUR_PUBLIC_KEY>"`
+    -  `export MONGODB_ATLAS_PRIVATE_KEY="<YOUR_PRIVATE_KEY>"`
+    -  `export MONGODB_ATLAS_BASE_URL="https://cloud-dev.mongodb.com/"` 
+    -  `export TF_VAR_org_id=<YOUR_ORG_ID>` 
+3. Run from terminal (from repository root) the command `terraform init`
+4. Run from terminal (from repository root) the command `terraform test`
```

Comment:
[q] As tests are defined in separate files, do they run in parallel?

Reply:
No, they actually run sequentially like this:
1. Terraform runs a certain test (its run blocks) and creates the corresponding plan, resources, and state.
2. Once everything is created and the asserts are passed (i.e. the test pass correctly), Terraform destroys everything that was created within the test
3. Then, Terraform runs the next test

Reply:
thanks for the details, just saw they have an open issue as well related to this: https://github.com/hashicorp/terraform/issues/34180